### PR TITLE
[Docs] dashboard and ci docs

### PR DESCRIPTION
**Pull request by luarss**
@dralabeing @vvbandeira

### CODE REVIEW

**Comment by luarss** [comment]
image

https://openroad-flow-scripts--1384.org.readthedocs.build/en/1384/user/QoRDashboard.html
https://openroad-flow-scripts--1384.org.readthedocs.build/en/1384/user/Jenkins.html

File: docs/user/QoRDashboard.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Using the OpenROAD Dashboard to track and analyze Quality of Results
+
+```{warning}
+This page is under construction. Check back later!
+```
```
Comment (vvbandeira) [inline docs/user/QoRDashboard.md]: @habibayassin
@luarss is working on our documentation, and we would like to include the great work you have done with the new dashboard. Could you help Jack write a small description of your work and create a user guide to showcase the currently deployed features?

Edit: I forgot to mention we can include a walkthrough like the above with Jenkins using a Google Slide presentation so we can keep images up to date easily.

Comment (habibayassin) [reply docs/user/QoRDashboard.md]: @vvbandeira yes of course... @luarss I will send you an email with these details.

**Comment by luarss** [comment]
@habibayassin @vvbandeira Added the dashboard docs.

I think this will do for now, maybe @habibayassin can work on a Google slides version in due time

File: docs/user/Flow_QoR_CI.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# OpenROAD QoR and CI Metrics
+
+## OpenROAD Dashboard 
+
+The [OpenROAD Dashboard](QoRDashboard.md) provides an easy way to explore 
+Quality-of-Results (QoR) over time and describes flow-QoR metrics, such as:
```
Comment (vvbandeira) [inline docs/user/Flow_QoR_CI.md]:
```suggestion
Quality-of-Results (QoR) over time and describes flow-QoR metrics, such as
```

File: docs/user/Flow_QoR_CI.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+timing, area, power, and so on. 
+
+## Jenkins CI
+
+We use [Jenkins](Jenkins.md) as our continuous integration (CI) tool,
+you may use it to find out metrics regarding regression tests.
```
Comment (vvbandeira) [inline docs/user/Flow_QoR_CI.md]:
```suggestion
We use [Jenkins](Jenkins.md) as our continuous integration (CI) tool.
You may use it to find out metrics regarding regression tests. 
```

File: docs/user/Jenkins.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Jenkins Continuous Integration
+
+For an in-depth tutorial on how to access the CI metrics, please see
+the [link](https://docs.google.com/presentation/d/1kWHLjUBFcd0stnDaPNi_pt9WFrrsR7tQ95BGhT1yOvw/)
```
Comment (vvbandeira) [inline docs/user/Jenkins.md]:
```suggestion
# Jenkins Continuous Integration

The Jenkins website is https://jenkins.openroad.tools/. For an in-depth tutorial on how to access the CI metrics, please see
```

File: docs/user/QoRDashboard.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+## Comparison 
+
+The comparison page enables users to compare metrics between branches 
+or commits for a specific platform and its designs. The comparison is 
+shown in the form of a table that showcases the differences between the 
+base source and the test source. This page also offers deeper insights 
+like the platform summary table that presents statistical trends for 
+different designs within a platform. As well as, a summary table that 
+provides insights for the set of selected designs, with added 
+functionality for filtering.
```
Comment (vvbandeira) [inline docs/user/QoRDashboard.md]:
```suggestion
The comparison page lets users compare metrics between branches 
or commits for a specific platform and its designs. The comparison is 
shown as a table showcasing the differences between the 
base source and the test source. This page also offers deeper insights, 
like the platform summary table that presents statistical trends for 
different designs within a platform and a summary table that 
provides insights for the set of selected designs, with added 
functionality for filtering. 
```

File: docs/user/QoRDashboard.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Using the OpenROAD Dashboard to track and analyze Quality of Results
+
+The metrics dashboard is tailored for RTL-to-GDS flow stages, 
+targeting different designs across various platforms. The dashboard 
+comprises two main pages: the trends page and the comparison page.
```
Comment (vvbandeira) [inline docs/user/QoRDashboard.md]:
```suggestion
# Using the OpenROAD Dashboard to Track and Analyze Quality of Results

The metrics dashboard (https://dashboard.theopenroadproject.org/)
is tailored for RTL-to-GDS flow stages, targeting different designs across
various platforms. The dashboard comprises two main pages: the trends
page and the comparison page. 
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Comment by vvbandeira** [comment]
Also, this is a more interesting picture of the metrics trend. Please convert to webp, I cannot upload webp images.

dashboard

File: docs/user/Flow_QoR_CI.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# OpenROAD QoR and CI Metrics
+
+## OpenROAD Dashboard 
+
+The [OpenROAD Dashboard](QoRDashboard.md) provides an easy way to explore 
+Quality-of-Results (QoR) over time and describes flow-QoR metrics, such as:
+timing, area, power, and so on.
```
Comment (dralabeing) [inline docs/user/Flow_QoR_CI.md]: change "and so on" to etc.

**Review by dralabeing — COMMENTED** [review]
Include the definition and use of "View Rules" for trend tracking w.r.t CI limit or dashboard as applicable explaining the use model.

**Comment by luarss** [comment]
> Include the definition and use of "View Rules" for trend tracking w.r.t CI limit or dashboard as applicable explaining the use model.

@habibayassin Can you help with this comment? Thanks.

**Comment by vvbandeira** [comment]
> > Include the definition and use of "View Rules" for trend tracking w.r.t CI limit or dashboard as applicable explaining the use model.
> 
> @habibayassin Can you help with this comment? Thanks.

No need. We will change the name to "Show CI Limits", this will make the use clear.

**Comment by vvbandeira** [comment]
@luarss 
Please update the image to reflect the latest change in the toggle name:

image

File: docs/user/Flow_QoR_CI.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+## OpenROAD Dashboard 
+
+The [OpenROAD Dashboard](QoRDashboard.md) provides an easy way to explore 
+Quality-of-Results (QoR) over time and describes flow-QoR metrics, such as timing, area, power, etc. 
+
+## Jenkins CI
+
+We use [Jenkins](Jenkins.md) as our continuous integration (CI) tool.
+You may use it to find out metrics regarding regression tests.
```
Comment (vvbandeira) [inline docs/user/Flow_QoR_CI.md]:
```suggestion
We use Jenkins as our continuous integration (CI) tool.
You may use it to find out metrics regarding regression tests.
See more about accessing and taking advantage of our CI [here](Jenkins.md).
```

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**